backend/inspirehep/records/serializers/json/literature.py

```diff
@@ -31,6 +32,16 @@
 
 # Literature
 literature_json = ConditionalMultiSchemaJSONSerializer(
+    [
+        (
+            lambda _: is_superuser_or_cataloger_logged_in(),
+            wrap_schema_class_with_metadata(LiteratureAdminSchema),
+        ),
+        (None, wrap_schema_class_with_metadata(LiteraturePublicListSchema)),
+    ]
+)
+
+literature_json_detailed = ConditionalMultiSchemaJSONSerializer(
```

Comment:
in some places it's `_detailed` in some others is `_record`, let unify it 

```collection_json``` -> for record response
```collection_json_search``` -> for search response

WDYT?